src/java.desktop/macosx/classes/sun/awt/CGraphicsConfig.java

```diff
@@ -80,7 +80,7 @@ public AffineTransform getNormalizingTransform() {
      * Creates a new SurfaceData that will be associated with the given
      * CGLLayer.
      */
```

Comment:
I guess we need to specify CGLLayer/MTLLayer as now we are implementing this method in both pipeline.

Reply:
I have updated JDK-8263363 to include this.